plugins/Vestige/Vestige.cpp

```diff
@@ -583,11 +583,12 @@ VestigeInstrumentView::VestigeInstrumentView( Instrument * _instrument,
 
 	m_selPresetButton->setMenu(menu);
 
+	constexpr int buttonFontSize = 12;
```

Comment:
Why this tho? Wouldn't it be better to use 12 directly in places instead of this variable?

Reply:
This is to make the font size of all push buttons on the Vestige GUI consistent. If the sizes need to be changed it can be done in one place.

Reply:
Ok then.

Reply:
You can resolve this